flintrock/flintrock.py

```diff
@@ -172,8 +175,11 @@ def cli(cli_context, config, provider):
 @click.option('--spark-version',
               help="Spark release version to install.")
 @click.option('--spark-git-commit',
-              help="Git commit hash to build Spark from. "
-                   "--spark-version and --spark-git-commit are mutually exclusive.")
+              help="Git commit hash to build Spark from."
+                   "Can be \"latest\" to build Spark from the latest available commit."
```

Comment:
The trailing space I had in there previously was intentional. ;)

These strings need trailing spaces, otherwise when they are joined each sentence will be glued to the previous one.